### ISSUE TITLE
TableLayout integration in LibGdx 1.2.0 breaks Value.percentWidth

### ISSUE THREAD

**Issue by elipsitz** [libgdx/libgdx#2016]
Previously, TableLayout was in EsotericSoftware's github repository. When 1.2.0 came along, it was integrated more directly in 1.1.0. However, this changed the function of Value.percentWidth (and percentHeight). I believe this change was accidental, but either way, it was not documented.

The EsotericSoftware version of Value.percentWidth lists the following description:
https://github.com/EsotericSoftware/tablelayout/blob/master/tablelayout/src/com/esotericsoftware/tablelayout/Value.java#L170
/*\* Returns a value that is a percentage of the table's width. */
Therefore, if you did something like, table.add(actor).width(Value.percentWidth(0.1f)); you could expect the widget to take up 10% of the table's width.

However, in the new version, Value.percentWidth does the following:
https://github.com/libgdx/libgdx/blob/master/gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Value.java#L95
/*\* Returns a value that is a percentage of the actor's width. */
Meaning that it returns a percentage of the actor specified in get(Actor actor)'s width.
This is perfectly fine, however, Table.java does not correctly use this modified Value.

This is evident here: https://github.com/libgdx/libgdx/blob/master/gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Table.java#L802

Value.get(...) should be called with a reference to the table (like Value.get(table) ) in order to preserve the previous behavior. However, Table does Value.get(a), where a is the actor in the cell. This makes no sense whatsoever. 

Basically, the issue here is that Table is incorrectly using Value, and it defeats the purpose of using Value.percentWidth(...).
